local-volume/provisioner/README.md

```diff
@@ -0,0 +1,53 @@
+# local-volume-provisioner
+
+NOTE: provisioner is still in development and is not functional yet
+
+local-volume-provisioner is an out-of-tree static provisioner for the Local volume plugin, which is a 1.7 alpha feature.
+
+It runs on each node in the cluster and monitors specified directories to look for new local mount points.  It then statically creates a Local PersistentVolume for each mount point.  It also monitors when the PersistentVolumes have been deleted, and will clean up the volume.
```

Comment:
Yeah, I mean released, not deleted.  If it's deleted manually by the user, then we won't do any cleanup.

If the admin wants to delete the underlying storage, then I think the best way to do it would be:
1. Stop the pod that is using the volume
2. Admin removes the volume
3. Delete the PVC
4. Daemonset will try to cleanup, and it fails since the volume is no longer there.  But will still delete the API object.